http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3872,6 +3926,42 @@ components:
           type: string
           description: Optional id of source workspace if app is a clone.
 
+    UpdateAppsRequest:
+      description: a request to update a specific set of apps (v1 or v2)
+      required:
+        - appType
+        - cloudProvider
+      type: object
+      properties:
+        appType:
+          $ref: '#/components/schemas/AppType'
+        cloudProvider:
+          $ref: '#/components/schemas/CloudProvider'
+        appVersionsInclude:
+          type: array
+          description: If provided, apps currently running these chart versions will be updated. Applied after a config-level default exclusion list.
+          items:
+            type: string
+        appVersionsExclude:
+          type: array
+          description: If provided, apps currently running these chart versions will NOT be updated. Applied after a config-level default exclusion list.
+          items:
+            type: string
+        googleProject:
+          type: string
+          description: If provided, update apps in this Google project
+        workspaceId:
+          type: string
+          description: If provided, update apps in this workspace
+        appNames:
+          type: array
+          description: If provided, apps with these names will be updated
+          items:
+            type: string
+        dryRun:
+          type: boolean
+          description: If true, report on what apps would be updated but take no action. Defaults to false.
```

Comment:
@jdcanas the dryRun option is explained here, is that enough or you would rather have it explained somewhere else in the code as well?